lib/node_modules/@stdlib/stats/base/nanmskmax/test/test.ndarray.js

```diff
@@ -164,16 +221,96 @@ tape( 'the function supports `stride` parameters', function test( t ) {
 		0
 	];
 
-	N = floor( x.length / 2 );
-	v = nanmskmax( N, x, 2, 0, mask, 2, 0 );
+	v = nanmskmax( 5, x, 2, 0, mask, 2, 0 );
+
+	t.strictEqual( v, 4.0, 'returns expected value' );
+	t.end();
+});
+
+tape( 'the function supports `stride` parameters (accessors)', function test( t ) {
+	var mask;
+	var x;
+	var v;
+
+	x = [
+		1.0,  // 0
+		2.0,
+		2.0,  // 1
+		-7.0,
+		-2.0, // 2
+		3.0,
+		4.0,  // 3
+		2.0,
+		5.0,  // 4
+		6.0,
+		NaN,  // 5
+		NaN
+	];
+	mask = [
+		0, // 0
+		0,
+		0, // 1
+		0,
+		0, // 2
+		0,
+		0, // 3
+		0,
+		1, // 4
+		1,
+		0, // 5
+		0
+	];
+
+	x = toAccessorArray( x );
+
+	v = nanmskmax( 5, x, 2, 0, toAccessorArray( mask ), 2, 0 );
```

Comment:
```suggestion
	v = nanmskmax( 6, x, 2, 0, toAccessorArray( mask ), 2, 0 );
```
follow the commented indices given above

Reply:
same comment for all other instances